### ISSUE TITLE
False positive: com.rpl.specter/ALL in cljs is marked as an "Unresolved var"

### ISSUE THREAD

**Issue by kovasap** [clj-kondo/clj-kondo#2557]
- [ x ] I have read the [Clojure etiquette](https://clojure.org/community/etiquette) and will respect it when communicating on this platform.

**version**

```
clj-kondo --version                                                             
clj-kondo v2025.06.05
```

**platform**

arch linux, but i think this problem is platform agnostic.  I see the issue when running as part of clojure-lsp and directly on the command line.

**problem**

I get an undefined var when linting https://github.com/kovasap/draft-concept/blob/f8ba1c763745123ef0b74711ee9007417f7be702/src/main/app/interface/world_map.cljs#L36 with  `clj-kondo --lint src/main/app/interface/world_map.cljs`.  It is not actually an undefined var in that my code runs fine.

**Comment by borkdude** [comment]
Have you added this configuration: https://github.com/redplanetlabs/specter?tab=readme-ov-file#clj-kondo
and also linted the specter dependency? When doing both these things, the warning should be gone.

**Comment by kovasap** [comment]
Nope, I just independently discovered that config and was coming here to post it and close the issue.  It fixes the problem.

Sorry for the noise!

**Comment by amiorin** [comment]
I still have problems.

My config
```clojure
{:lint-as {big-config.utils/assert-args-present clojure.core/->
           big-config.utils/debug clojure.core/def
           com.rpl.specter/recursive-path clojure.core/as->
           com.rpl.specter/defcollector clojure.core/defn
           com.rpl.specter/defdynamicnav clojure.core/defn
           com.rpl.specter/defmacroalias clojure.core/def
           com.rpl.specter/defnav clojure.core/defn
           com.rpl.specter/defrichnav clojure.core/defn}}
```

<img width="1067" height="250" alt="Image" src="https://github.com/user-attachments/assets/ff7f6d3f-31f6-4765-8921-0a4112677fbe" />

**Comment by borkdude** [comment]
@amiorin If you can put together a Github repo that I can clone locally, with instructions how to reproduce the problem using the command line, I will take a look.

**Comment by amiorin** [comment]
@borkdude 
I cannot reproduce the problem anymore. Maybe some caching issue.
